packages/grafana-ui/src/components/BarGauge/BarGauge.tsx

```diff
@@ -371,6 +392,11 @@ export function calculateBarAndValueDimensions(props: Props): BarAndValueDimensi
     valueHeight = height - titleDim.height;
     valueWidth = Math.max(Math.min(width * 0.2, MAX_VALUE_WIDTH), realValueWidth);
```

Comment:
This could be an if...else so we don't do these calculations if `valueMode` is hidden